easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,40 @@
+name = 'NWChem'
+version = '6.6.revision27746'
+verdate = '2015-10-20'
+pyver = '2.7.11'
+versionsuffix = '-%s-Python-%s' % (verdate, pyver)
```

Comment:
@RvDijk please make the easyconfig filename match this versionsuffix (`2015-10-20`  is missing)